docs/application-connector/docs/044-remote-evironment-custom-resource.md

```diff
@@ -29,9 +29,27 @@ spec:
 
 This table lists all the possible parameters of a given resource together with their descriptions:
 
-
 | Parameter   |      Mandatory      |  Description |
 |:----------:|:-------------:|:------|
 | **metadata.name** |    **YES**   | Specifies the name of the CR. |
-| **spec.description** |    **NO**   | Provides a short, human-readable description of the RE. |
+| **spec.source** |    **NO**   | Identifies the Remote Environment in the cluster. |
+| **spec.description** |    **NO**   | Describes the connected Remote Environment.  |
+| **spec.accessLabel** |    **NO**   | Labels the RE when an EnvironmentMapping is created. |
 | **spec.labels** |    **NO**   | Defines the labels of the RE. |
+| **spec.services** |    **NO**   | Contains all services that the Remote Environment provides. |
+| **spec.services.id** |    **YES**   | Identifies the service that the Remote Environment provides. |
+| **spec.services.identifier** |    **NO**   | Provides an additional identifier of the ServiceClass. |
+| **spec.services.name** |    **NO**   | Represents a unique name of the service used by the Service Catalog. |
+| **spec.services.displayName** |    **YES**   | Specifies a human-readable name of the Remote Environment service. Parameter provided by the Metadata Service, do not edit. |
+| **spec.services.description** |    **NO**   | Provides a short, human-readable description of the service offered by the RE. Parameter provided by the Metadata Service, do not edit. |
+| **spec.services.longDescription** |    **NO**   | Provides a longer, human-readable description of the service offered by the RE. Parameter provided by the Metadata Service, do not edit. |
+| **spec.services.providerDisplayName** |    **YES**   | Specifies a human-readable name of the Remote Environment service provided. Parameter provided by the Metadata Service, do not edit. |
+| **spec.services.tags** |    **NO**   | Specifies additional tags used for better documentation of the available APIs. Parameter provided by the Metadata Service, do not edit. |
+| **spec.services.labels** |    **NO**   | Specifies additional labels for the service offered by the RE. Parameter provided by the Metadata Service, do not edit. |
+| **spec.services.entries** |    **YES**   | Contains the information about the APIs and Events that the service offered by the RE provides. Parameter provided by the Metadata Service, do not edit. |
+| **spec.services.entries.type** |    **YES**   | Specify the entry type: API or Event. Parameter provided by the Metadata Service, do not edit. |
```

Comment:
`Specifies whether the entry is of API or Event type.`  ? -> for sure you should be consistent and use 3rd form of the verb at the beginning